spec/spec_helper_acceptance.rb

```diff
@@ -22,6 +22,6 @@
   c.before :suite do
     # Install module dependencies
     on hosts, puppet('module', 'install', 'puppetlabs-stdlib'), { :acceptable_exit_codes => [0,1], :run_in_parallel => true }
-    on hosts, puppet('module', 'install', 'puppetlabs-apt', '--version', '">= 4.0.0 < 5.0.0"'), { :acceptable_exit_codes => [0,1], :run_in_parallel => true }
+    on hosts, puppet('module', 'install', 'puppetlabs-apt', '--version', '">= 5.0.1"'), { :acceptable_exit_codes => [0,1], :run_in_parallel => true }
```

Comment:
We should have an upper bound here and the documentation to show `">= 5.0.1 < 6.0.0"`